src/connector.ts

```diff
@@ -144,18 +144,28 @@ export class Connector {
 	}
 
 	private setupReloadDataKoaEndpoint(): void {
-		this.koaRouter.get('/reloadData/:rundownName', async (ctx, next): Promise<void> => {
-			const RUNDOWN_EXTERNAL_ID_SUFFIX = '_1'
+		const KOA_PORT: number = 3007
+		const RUNDOWN_EXTERNAL_ID_SUFFIX: string = '_1'
+
+		this.koaRouter.post('/reloadData/:rundownName', async (context, next): Promise<void> => {
```

Comment:
This is not following REST standards (maybe that is fine).
If we want this URL to follow REST standards, it should be changed to do so.

Personally, I would prefer it, but I can see arguments for not doing it.

Reply:
I see your point, and on Sofie Server i'd say it makes a lot of sense. Here, however,  i might counter argue that because of the setup where we both actually work on playlists and rundowns to do this operation, it's difficult to just use e.g. `rundowns/reloadData/:rundownName` since it is used for both playlists and rundowns.

Reply:
On another note, this is tied to our implementation of Sofie Server alone, so i will name it to fit the standard it follows!